LeanMapperQuery/Query.php

```diff
@@ -446,13 +455,13 @@ public function cast($entityClass)
 		return $this;
 	}
 
-
 	/**
 	 * @inheritdoc
 	 * @throws     InvalidArgumentException
 	 */
-	public function applyQuery(Fluent $fluent, IMapper $mapper)
+	public function applyQuery(Fluent $fluent, IMapper $mapper, $primaryTable = NULL)
 	{
+		$this->primaryTable = $primaryTable;
```

Comment:
Potřebujeme další proměnnou? Nestačí hodnotu uložit do samotného `sourceTableName`?